android/KMAPro/kMAPro/build.gradle

```diff
@@ -55,7 +55,7 @@ android {
 
     buildTypes {
         debug {
-            applicationIdSuffix ".debug"
+            // applicationIdSuffix ".debug"
```

Comment:
In order to test, we have to use exactly the same `applicationId` of `com.tavultesoft.kmapro` as we find in the Play Store. If desired, we can back this change out before we merge the PR, after testing.